docker/pg_tests/scripts/tests/pgbackrest_backup_fetch_test.sh

```diff
@@ -0,0 +1,47 @@
+set -e -x
+CONFIG_FILE="/tmp/configs/pgbackrest_backup_fetch_config.json"
+COMMON_CONFIG="/tmp/configs/common_config.json"
+TMP_CONFIG="/tmp/configs/tmp_config.json"
+PGBACKREST_CONFIG="/tmp/configs/pgbackrest_backup_fetch_config.ini"
+cat ${CONFIG_FILE} > ${TMP_CONFIG}
+echo "," >> ${TMP_CONFIG}
+cat ${COMMON_CONFIG} >> ${TMP_CONFIG}
+/tmp/scripts/wrap_config_file.sh ${TMP_CONFIG}
+ 
+
+/usr/lib/postgresql/10/bin/initdb ${PGDATA}
+
+archive_command="/usr/bin/timeout 600 pgbackrest --stanza=main --pg1-path=${PGDATA} --repo1-path=/tmp/pgbackrest-backups archive-push %p"
+echo "archive_mode = on" >> ${PGDATA}/postgresql.conf
+echo "archive_command = '${archive_command}'" >> ${PGDATA}/postgresql.conf
+echo "archive_timeout = 600" >> ${PGDATA}/postgresql.conf
+
+/usr/lib/postgresql/10/bin/pg_ctl -D ${PGDATA} -w start
+/tmp/scripts/wait_while_pg_not_ready.sh
+
+mkdir -m 770 /tmp/pgbackrest-backups
+
+pgbackrest --stanza=main --pg1-path=${PGDATA} --repo1-path=/tmp/pgbackrest-backups stanza-create
```

Comment:
Why not to use the `repo1-s3-*` pgbackrest settings instead of manual copy?

Reply:
Pgbackrest requires minio with https. We can change minio settings and enable https or we can copy backup to minio with s3cmd